scm/driver/github/repo.go

```diff
@@ -53,6 +53,11 @@ type hook struct {
 	} `json:"config"`
 }
 
+type RepositoryListResponse struct {
```

Comment:
should start with a lowercase letter

Reply:
agree this should be lowercase.   Also we do not suffix with `Response` so it should be `repositoryList` instead

Reply:
changed